.bin/bashbrew-buildkit-env-setup.sh

```diff
@@ -47,7 +47,7 @@ _bashbrew_buildkit_env_setup() {
 	local vars='{}'
 
 	local dockerfileTag
-	dockerfileTag="$(grep <<<"$externalPins" -m1 '^docker/dockerfile:')"
+	dockerfileTag="$(grep <<<"$externalPins" -m1 '^tianon/buildkit:')"
```

Comment:
Why is it called `tianon/buildkit` and not `tianon/dockerfile`?

Reply:
That was the genesis of https://github.com/tianon/dockerfiles/pull/733 -- it's a single image that includes _both_ buildkitd and the dockerfile frontend, and uses context clues to decide which one to run automatically.  The justification is that they're built from the same exact source tree, and the "dockerfiles" tags/releases have thus far been a strict subset of the buildkit tags/releases (which makes sense), so it's much easier to apply patches consistently if we build both from the same patched source tree, and at that point it's even easier to both build _and_ use if they're the same image (it's really easy IMO to remember that `tianon/buildkit` is _the_ buildkit image, no matter which context I'm needing a buildkit-related image in).

Reply:
To perhaps illustrate what I mean about the tags better:

```console
$ git ls-remote --tags https://github.com/moby/buildkit.git | grep 0865fcc9b78559e856e81dc52b3613701e7be28d
0865fcc9b78559e856e81dc52b3613701e7be28d	refs/tags/dockerfile/1.10.0
0865fcc9b78559e856e81dc52b3613701e7be28d	refs/tags/dockerfile/1.10.0-labs
0865fcc9b78559e856e81dc52b3613701e7be28d	refs/tags/v0.16.0

$ git tag -l | grep -E '^v0[.](1[3-6])[.]' | xargs -r git rev-parse | sort -u | xargs -rn1 git log -1 --oneline --decorate --format='format:%S %d'
0865fcc9b78559e856e81dc52b3613701e7be28d  (tag: v0.16.0, tag: dockerfile/1.10.0-labs, tag: dockerfile/1.10.0)
1981eb123dc979fc71d097adeb5bbb84110aa9f4  (tag: v0.13.0-beta3)
2ae42e0c0c793d7d66b7a23424af6fd6c2f9c8f3  (tag: v0.13.1)
2afc050d57d17983f3f662d5424c2725a35c60f4  (tag: v0.13.0, tag: dockerfile/1.7.0-labs, tag: dockerfile/1.7.0)
2e18d709fefdcc2db20853ee241c75b058189d39  (tag: v0.13.2, tag: dockerfile/1.7.1-labs, tag: dockerfile/1.7.1, origin/v0.13, v0.13)
331b5d5a4b0d5980e091eaf50f2bedd253cc8cc9  (tag: v0.13.0-rc1)
4d9a4e5df9e11596a3261c1952cde3c6346be762  (tag: v0.14.0, tag: dockerfile/1.8.0-labs, tag: dockerfile/1.8.0)
596ef8f01e11e15889576a88ffa4c7f92fa44518  (tag: v0.13.0-rc2, tag: dockerfile/1.7.0-rc1-labs, tag: dockerfile/1.7.0-rc1)
7b23ff0766d3c639a9a0ff9ff0af630089a12eb8  (tag: v0.13.0-rc3)
8ade9b239b580ff2af97b527e2484df60d6a2b6b  (tag: v0.15.0-rc1, tag: dockerfile/1.9.0-rc1-labs, tag: dockerfile/1.9.0-rc1)
979542e90f2cb38077c808e0867d8d2c16ed10b8  (tag: v0.15.1)
9e14164a1099d3e41b58fc879cbdd6f2b2edb04e  (tag: v0.15.2, origin/v0.15)
aebcc1f0eabcbaeef4be8e948641f653140fe2bf  (tag: v0.14.0-rc2, tag: dockerfile/1.8.0-rc2-labs, tag: dockerfile/1.8.0-rc2)
b9fcd7688a5d7d4461f0c1706216b6d4b3e584e6  (tag: v0.13.0-beta1)
c958c686e9ebf6b7d754881b92ba23b7859c5e59  (tag: v0.16.0-rc1, tag: dockerfile/1.10.0-rc1-labs, tag: dockerfile/1.10.0-rc1)
c9d08dde729d5ff0c7f6faf38b5bfd9ce8a753d5  (tag: v0.16.0-rc2)
e740d4b90ab98e1a10f3aae58d421bf135cff3cf  (tag: v0.14.0-rc1, tag: dockerfile/1.8.0-rc1-labs, tag: dockerfile/1.8.0-rc1)
e83d79a51fb49aeb921d8a2348ae14a58701c98c  (tag: v0.15.0-rc2, tag: v0.15.0, tag: dockerfile/1.9.0-rc2-labs, tag: dockerfile/1.9.0-rc2, tag: dockerfile/1.9.0-labs, tag: dockerfile/1.9.0)
eb864a84592468ee9b434326cb7efd66f58555af  (tag: v0.14.1, tag: dockerfile/1.8.1-labs, tag: dockerfile/1.8.1, origin/v0.14)
ef61ad133d7fd20632eb964dfc78292360f2f09d  (tag: v0.13.0-beta2)
```